### PR TITLE
Fix @entangle reference typos on alpine-js.blade.php

### PR DESCRIPTION
**Pull request by taylormaguire**
Fix @@entangle references with @entangle

### CODE REVIEW

File: resources/views/docs/2/alpine-js.blade.php [hunk 1]
```diff
@@ -144,7 +144,7 @@ public function increment()
 $wire.__instance
 @endcomponent
 
-## Sharing State Between Livewire And Alpine: `@@entangle`
```
Comment (nuernbergerA) [inline resources/views/docs/2/alpine-js.blade.php:147]: That's valid for escape

File: resources/views/docs/2/alpine-js.blade.php [hunk 2]
```diff
@@ -258,7 +258,7 @@ public function delete()
 @endverbatim
 @endcomponent
 
-There are LOTS of different ways to use this utility, but one common example is using it in conjunction with the aforementioned `@@entangle` directive:
```
Comment (nuernbergerA) [inline resources/views/docs/2/alpine-js.blade.php:261]: That's valid for escape

File: resources/views/docs/2/alpine-js.blade.php [hunk 3]
```diff
@@ -282,7 +282,7 @@ public function delete()
 @endverbatim
 @endcomponent
 
-> Note: If the `.defer` modifier is passed via `wire:model.defer`, the `@@entangle` directive will automatically recognize it and add the `@entangle('...').defer` modifier under the hood.
```
Comment (nuernbergerA) [inline resources/views/docs/2/alpine-js.blade.php:285]: That's valid for escape

File: resources/views/docs/2/alpine-js.blade.php [hunk 4]
```diff
@@ -172,7 +172,7 @@ public function delete()
 @endslot
 @slot('view')
 @verbatim
-<div x-data="{ open: @@entangle('showDropdown') }">
```
Comment (nuernbergerA) [inline resources/views/docs/2/alpine-js.blade.php:175]: That's the actual bug

**Review by nuernbergerA — CHANGES_REQUESTED** [review]

**Comment by taylormaguire** [comment]
All good, Caleb fixed it

**Pull request closed without merging**